flags/src/test/java/com/yahoo/vespa/flags/custom/ClusterCapacityTest.java

```diff
@@ -38,4 +39,29 @@ void serialization2() throws IOException {
         assertEquals(2.3, deserialized.bandwidthGbps(), 0.0001);
         assertEquals(7, deserialized.count());
     }
+
+    @Test
+    void serializationWithNoNodeResources() throws IOException {
+        ClusterCapacity clusterCapacity = new ClusterCapacity(7, null, null, null, null);
+        ObjectMapper mapper = new ObjectMapper();
+        String json = mapper.writeValueAsString(clusterCapacity);
+        assertEquals("{\"count\":7}", json);
+
+        ClusterCapacity deserialized = mapper.readValue(json, ClusterCapacity.class);
+        assertEquals(0.0, deserialized.vcpu(), 0.0001);
+        assertEquals(0.0, deserialized.memoryGb(), 0.0001);
+        assertEquals(0.0, deserialized.diskGb(), 0.0001);
+        assertEquals(1.0, deserialized.bandwidthGbps(), 0.0001);  // 1.0 is used as fallback
+        assertEquals(7, deserialized.count());
+
+        // Test that no values for node resources is allowed in json also
```

Comment:
The second below seems to be the same as the above, `json == jsonWithNoValuesForNodeResources == {"count":7}`, so both the `deserialized` are identical.

Reply:
Yes, you are right, went back and forth and forgot to remove this. Fixing